qdrant-landing/content/documentation/headless/snippets/cloud-inference/vector-search/upload-data/csharp.md

```diff
@@ -0,0 +1,36 @@
+```csharp
+var denseModel = "sentence-transformers/all-minilm-l6-v2";
+var bm25Model = "qdrant/bm25";
+var dataset = LoadDataset("miriad/miriad-4.4M", "train[0:100]");
```

Comment:
The implementation of `LoadDataset` is upto the user to keep the snippets concise.